addons/rose/app/styles/rose/components/form/_radio-card.scss

```diff
@@ -180,18 +196,20 @@ $radio-svg: (
 
   &.error {
     .rose-form-radio-field {
-      &:not(:disabled):not(:focus) {
-        &:not(:disabled):not(:focus) {
+      &:not(:disabled, :focus) {
+        &:not(:disabled, :focus) {
```

Comment:
Are these duplicated?  Not sure why they were in the first place, but probably unnecessary?

Reply:
Indeed, seems duplicate and unneccesary states. I tested within storybook and I couldn't reproduce to show this specific not diabled and not focus.

Reply:
Looks like it, removed it